wiki/tests/testdata/models.py

```diff
@@ -8,3 +8,7 @@ class CustomUser(AbstractUser):
         some_field = models.IntegerField(default=0)
 except ImportError:
     pass
+
+
+class CustomGroup(models.Model):
```

Comment:
Yeah, maybe just leave it like that and then we'll discover once we impose requirements for a Group model, so we can fix the test and document it at the same time.....

@fritz-k do you feel like adding a doc section about the Group model ? :)